engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/transform/FormResourceTransformer.java

```diff
@@ -62,7 +62,8 @@ public Either<Failure, Void> transformResource(
                     .map(
                         noDuplicates -> {
                           final FormMetadataRecord formRecord = deployment.formMetadata().add();
-                          appendMetadataToFormRecord(formRecord, formId, resource);
+                          appendMetadataToFormRecord(
+                              formRecord, formId, resource, deployment.getTenantId());
                           writeFormRecord(formRecord, resource);
```

Comment:
❓ What do you mean? It's added to the record on line 119?

Reply:
I meant line 67 doesn't get the tenant id, but there's a side effect that changes the formRecord instead of returning a changed value. I was confused by this.

👍 all good